restraint.spec

```diff
@@ -341,6 +345,7 @@ fi
 
 %files rhts
 %defattr(-,root,root,-)
+%attr(0644, root, root) %config(missingok) %{_sharedstatedir}/restraint/install_config
```

Comment:
```suggestion
%attr(0644, root, root)%config(noreplace)/var/lib/%{name}/install_config
```

Reply:
I was going to get rid of missingok as it doesn't give us much.  I have more changes coming.  Yesterday I was working from a workspace where my changes didn't seem to take affect.  I moved to a new workspace and I've got a changeset I think you will prefer.